game/operation/operation.py

```diff
@@ -485,14 +484,15 @@ def generate_lua(
                     elif hasattr(flightTarget, "name"):
                         flightTargetName = flightTarget.name
                         flightTargetType = flightType + " TGT (Airbase)"
-                    luaData["TargetPoints"][flightTargetName] = {
+                    luaData["TargetPoints"][flight_count] = {
                         "name": flightTargetName,
                         "type": flightTargetType,
                         "position": {
                             "x": flightTarget.position.x,
                             "y": flightTarget.position.y,
                         },
                     }
+                    i += 1
```

Comment:
Am I reading this right? It looks like this is the `i` from the JTAC loop on L458, and that it does nothing. It looks like you wanted to increment `flight_count`?

Reply:
Good catch! THANK You!
I have not tested this one 100% yet, thats why i left it as draft :)